src/data/footerLinks.json

```diff
@@ -0,0 +1,87 @@
+{
+    "links": [
+        {
+            "title": "Useful links",
+            "items": [
+                {
+                    "label": "Helpshift Home",
+                    "to": "https://helpshift.com"
+                },
+                {
+                    "label": "About us",
+                    "to": "https://www.helpshift.com/company/"
+                },
+                {
+                    "label": "Github",
+                    "to": "https://github.com/helpshift"
+                },
+                {
+                    "label": "Helpshift Engineering",
+                    "to": "https://medium.com/helpshift-engineering"
+                },
+                {
+                    "label": "Knowledge Base",
+                    "to": "https://support.helpshift.com"
+                },
+                {
+                    "label": "Helpshift Blog",
+                    "to": "https://helpshift.com/blog"
+                }
+            ]
+        },
+        {
+            "title": "Get in touch",
+            "items": [
+                {
+                    "label": "LinkedIn",
+                    "to": "https://in.linkedin.com/company/helpshift"
+                },
+                {
+                    "label": "Twitter",
+                    "to": "https://twitter.com/helpshift"
+                },
+                {
+                    "label": "Facebook",
+                    "to": "https://www.facebook.com/helpshift/"
+                },
+                {
+                    "label": "Instagram",
+                    "to": "https://www.instagram.com/helpshift/?hl=en"
+                },
+                {
+                    "label": "YouTube",
+                    "to": "https://www.youtube.com/c/Helpshift"
+                },
+                {
+                    "label": "Contact Us",
```

Comment:
This is to contact from a sales/company perspective. We should also have a "report a bug" or contact us for support link. https://www.helpshift.com/company/contact-us/ can probably be replaced to cover those functionalities instead.

Reply:
We will be showing the Web Chat widget on this page. Maybe the "Report a bug" button can open the chat widget? I can add this as a backlog task

Reply:
Sure, please create a backlog and resolve.

Reply:
Added backlog task